src/for-developers/indexer.md

```diff
@@ -0,0 +1,13 @@
+# Fuel Indexer
+
+The Fuel Indexer is a standalone binary that can be used to index various components of the blockchain. These indexable components include blocks, transactions, and receipts and state within a fuel network, allowing for high-performance read-only access to the blockchain for advanced dApp use-cases.
+
+Events can be indexed by the Fuel Indexer by either using WASM modules, [as described in the Hello World example](https://fuellabs.github.io/fuel-indexer/master/examples/hello-indexer.html), or by writing native Rust handlers as described in the [Simple Native example](https://fuellabs.github.io/fuel-indexer/master/examples/simple-native.html).
```

Comment:
Hmmm I was under the impression that the Rust indexing was being phased out. cc @ra0x3

Reply:
We can remove any references to native execution for now, as we're primarily concerned with WASM execution only.